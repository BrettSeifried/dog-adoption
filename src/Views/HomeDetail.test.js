import { render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Home from './HomeDetail';

it('render all the dogs on a list', async () => {
  const { container } = render(
    <MemoryRouter>
      <Home />
    </MemoryRouter>
  );
  await screen.findByText('Rue');
  expect(container).toMatchSnapshot();
});
