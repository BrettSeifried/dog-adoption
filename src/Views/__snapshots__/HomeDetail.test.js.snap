// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render all the dogs on a list 1`] = `
<div>
  <ul>
    <div
      class="dog-card"
    >
      <a
        class="card-render"
        href="/dogs/1"
      >
        <h2
          class="name"
        >
          Rue
        </h2>
        <img
          alt="dog"
          src="https://placedog.net/1000/563?id=206"
        />
        <p>
          Rhodesian Ridge Back
        </p>
      </a>
      <a
        class="card-render"
        href="/dogs/3"
      >
        <h2
          class="name"
        >
          Toby
        </h2>
        <img
          alt="dog"
          src="https://placedog.net/1000/563?id=107"
        />
        <p>
          King Charles Spaniel
        </p>
      </a>
      <a
        class="card-render"
        href="/dogs/5"
      >
        <h2
          class="name"
        >
          Mel
        </h2>
        <img
          alt="dog"
          src="https://placedog.net/1000/563?id=36"
        />
        <p>
          Bull
        </p>
      </a>
      <a
        class="card-render"
        href="/dogs/7"
      >
        <h2
          class="name"
        >
          Chester
        </h2>
        <img
          alt="dog"
          src="https://placedog.net/1000/563?id=210"
        />
        <p>
          Poodle
        </p>
      </a>
      <a
        class="card-render"
        href="/dogs/6"
      >
        <h2
          class="name"
        >
          Lady Bird
        </h2>
        <img
          alt="dog"
          src="https://placedog.net/1000/563?id=6"
        />
        <p>
          Hound
        </p>
      </a>
      <a
        class="card-render"
        href="/dogs/4"
      >
        <h2
          class="name"
        >
          Pebbles
        </h2>
        <img
          alt="dog"
          src="https://placedog.net/1000/563?id=71
"
        />
        <p>
          Labrador
        </p>
      </a>
      <a
        class="card-render"
        href="/dogs/2"
      >
        <h2
          class="name"
        >
          Cato
        </h2>
        <img
          alt="dog"
          src="https://placedog.net/1000/563?id=199"
        />
        <p>
          Terrier
        </p>
      </a>
    </div>
  </ul>
</div>
`;
